packages/core-ethereum/src/types/accountEntry.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai'
+import BN from 'bn.js'
+import { Address } from '.'
+import Public from './public'
+import AccountEntry from './accountEntry'
+import Hash from './hash'
+import { stringToU8a } from '@hoprnet/hopr-utils'
+
+const EMPTY_ADDRESS = new Address(new Uint8Array({ length: Address.SIZE }))
+const EMPTY_PUBKEY = new Public(new Uint8Array({ length: Public.SIZE }))
+const EMPTY_SECRET = new Hash(new Uint8Array({ length: Hash.SIZE }))
+
+const partyAPubKey = Public.fromString('0x03362b7b26bddb151a03056422d37119eab3a716562b6c3efdc62dec1540c9b091')
```

Comment:
I noticed a few similar keys here and there, might it be worth having a separate constants file with these?